### PR TITLE
Create community-call-14-luglio

### PR DESCRIPTION
**Pull request by Francescopog**
## Description

This PR tackles:

* ...
* ...
* ...

In particular, the ...

## Checklist

* [ ] I followed the indications in [CONTRIBUTING.md](https://github.com/italia/developers.italia.it/blob/master/CONTRIBUTING.md)
* [ ] The documentation related to the proposed change has been updated accordingly (also comments in code).
* [ ] Have you written new tests for your core changes, as applicable?
* [ ] Have you successfully ran tests with your changes locally?
* [ ] Ready for review! :rocket:

## Fixes

Fixes #

### CODE REVIEW

**Comment by developers-italia-droid** [comment]
<table>
  <thead>
    <tr>
      <th width="50"></th>
      <th width="100%" data-danger-table="true">Fails</th>
    </tr>
  </thead>
  <tbody><tr>
      <td>:no_entry_sign:</td>
      <td>There is a problem with the commit message
> Create community-call-14-luglio
- subject may not be empty
- type may not be empty</td>
    </tr>
  </tbody>
</table>




<p align="right">
  Generated by :no_entry_sign: <a href="https://danger.systems/js">dangerJS</a> against 1a92e6b7d809ab04ddeee8123c1564669fde6423
</p>

File: _posts/it/community-call-14-luglio [hunk 1]
```diff
@@ -0,0 +1,17 @@
+
+layout: post
+title: "“Risorse per l’integrazione del Sistema Pubblico di Identità Digitale"
+subtitle: Disponibile il video della community call del 14 luglio sullo sviluppo di SDK per l’integrazione di SPID
+date: 2021-07-30 11:00:00 +0200
```
Comment (bfabio) [inline _posts/it/community-call-14-luglio:5]:
```suggestion
date: 2021-07-30 10:30:00 +0200
```

File: _posts/it/community-call-14-luglio [hunk 2]
```diff
@@ -0,0 +1,17 @@
...
+A concludere l’incontro, una sessione di domande e risposte come sempre ricca di spunti, idee e riflessioni, grazie agli interventi dei partecipanti.
+Dopo la pausa estiva, la community si ritroverà il prossimo **22 settembre**. Per sapere il tema della prossima call **resta in contatto con la community su [Slack](https://developersitalia.slack.com/archives/C023MSRP03V)** e **segui [Developer Italia](https://twitter.com/developersITA) e [Designers Italia])https://twitter.com/DesignersITA) anche su Twitter**.
+**Se vuoi rivedere la community call del 14 luglio, [guarda la registrazione su YouTube])https://www.youtube.com/watch?v=NoMUqVKK8Rc).**
```
Comment (bfabio) [inline _posts/it/community-call-14-luglio:17]:
```suggestion
**Se vuoi rivedere la community call del 14 luglio, [guarda la registrazione su YouTube])(https://www.youtube.com/watch?v=NoMUqVKK8Rc){:target="_blank"}.** 
```

File: _posts/it/community-call-14-luglio [hunk 3]
```diff
@@ -0,0 +1,17 @@
...
+date: 2021-07-30 11:00:00 +0200
+tags: developers opensource spid sdk
+categories: news
+author: Francesco Pognante
+image: /assets/icons/logo-it.png
+locale: it
+
+
+Altro giro, altra call: la community di Developers e Designers si è riunita nuovamente il **14 luglio** per parlare di **sviluppo di risorse (SDK)** per l’integrazione del Sistema Pubblico di Identità Digitale (SPID) nei servizi pubblici e privati. Sono intervenuti alcuni tra i contributori, manutentori e autori di alcune tra le più popolari SDK ad oggi disponibili su github italia.
+Nel corso della call, **Davide Brunato** ci ha raccontato il suo lavoro su spid-django, una app che semplifica l’integrazione SPID nei progetti basati su Django Framework. **Michele d’Amico** ha poi descritto la sua esperienza con l’integrazione di SPID tramite SimpleSAMLphp, la sua partecipazione alla Hackathon Developers e la creazione di spid-php. A seguire, **Roberto Gallea** ci ha illustrato spid-shibboleth-sp e spid-idp-proxy-shibboleth, **Luca Leonardo** Scorcia ha descritto le peculiarità di  spid-keycloak-provider e, infine, **Daniele Giallonardo** ci ha spiegato quali sono le caratteristiche di spid-aspnetcore.
```
Comment (bfabio) [inline _posts/it/community-call-14-luglio:14]:
```suggestion
Nel corso della call, **Davide Brunato** ci ha raccontato il suo lavoro su spid-django, una app che semplifica l’integrazione SPID nei progetti basati su Django Framework. **Michele d’Amico** ha poi descritto la sua esperienza con l’integrazione di SPID tramite SimpleSAMLphp, la sua partecipazione alla Hackathon Developers e la creazione di spid-php. A seguire, **Roberto Gallea** ci ha illustrato [spid-sp-shibboleth](https://github.com/italia/spid-sp-shibboleth){:target="_blank"} e [spid-idp-proxy-shibboleth](https://github.com/italia/spid-idp-proxy-shibboleth){:target="_blank"}, **Luca Leonardo Scorcia** ha descritto le peculiarità di  [spid-keycloak-provider](https://github.com/italia/spid-keycloak-provider){:target="_blank"} e, infine, **Daniele Giallonardo** ci ha spiegato quali sono le caratteristiche di  [spid-aspnetcore](https://github.com/italia/spid-aspnetcore){:target="_blank"}.
```

File: _posts/it/2021-07-30-community-call-14-luglio [hunk 4]
```diff
@@ -0,0 +1,17 @@
...
+date: 2021-07-30 10:30:00 +0200
+tags: developers opensource spid sdk
+categories: news
+author: Francesco Pognante
+image: /assets/icons/logo-it.png
+locale: it
+
+
+Altro giro, altra call: la community di Developers e Designers si è riunita nuovamente il **14 luglio** per parlare di **sviluppo di risorse (SDK)** per l’integrazione del Sistema Pubblico di Identità Digitale (SPID) nei servizi pubblici e privati. Sono intervenuti alcuni tra i contributori, manutentori e autori di alcune tra le più popolari SDK ad oggi disponibili su github italia.
+Nel corso della call, **Davide Brunato** ci ha raccontato il suo lavoro su spid-django, una app che semplifica l’integrazione SPID nei progetti basati su Django Framework. **Michele d’Amico** ha poi descritto la sua esperienza con l’integrazione di SPID tramite SimpleSAMLphp, la sua partecipazione alla Hackathon Developers e la creazione di spid-php. A seguire, **Roberto Gallea** ci ha illustrato spid-shibboleth-sp e spid-idp-proxy-shibboleth, **Luca Leonardo** Scorcia ha descritto le peculiarità di  spid-keycloak-provider e, infine, **Daniele Giallonardo** ci ha spiegato quali sono le caratteristiche di spid-aspnetcore.
```
Comment (bfabio) [inline _posts/it/2021-07-30-community-call-14-luglio:14]:
```suggestion
Nel corso della call, **Davide Brunato** ci ha raccontato il suo lavoro su spid-django, una app che semplifica l’integrazione SPID nei progetti basati su Django Framework. **Michele d’Amico** ha poi descritto la sua esperienza con l’integrazione di SPID tramite SimpleSAMLphp, la sua partecipazione alla Hackathon Developers e la creazione di spid-php. A seguire, **Roberto Gallea** ci ha illustrato [spid-sp-shibboleth](https://github.com/italia/spid-sp-shibboleth){:target="_blank"} e [spid-idp-proxy-shibboleth](https://github.com/italia/spid-idp-proxy-shibboleth){:target="_blank"}, **Luca Leonardo Scorcia** ha descritto le peculiarità di  [spid-keycloak-provider](https://github.com/italia/spid-keycloak-provider){:target="_blank"} e, infine, **Daniele Giallonardo** ci ha spiegato quali sono le caratteristiche di  [spid-aspnetcore](https://github.com/italia/spid-aspnetcore){:target="_blank"}.
```

File: _posts/it/2021-07-30-community-call-14-luglio.md [hunk 5]
```diff
@@ -0,0 +1,17 @@
+
```
Comment (bfabio) [inline _posts/it/2021-07-30-community-call-14-luglio.md:1]:
```suggestion
---
```

File: _posts/it/2021-07-30-community-call-14-luglio.md [hunk 6]
```diff
@@ -0,0 +1,17 @@
...
+layout: post
+title: "“Risorse per l’integrazione del Sistema Pubblico di Identità Digitale"
+subtitle: Disponibile il video della community call del 14 luglio sullo sviluppo di SDK per l’integrazione di SPID
+date: 2021-07-30 10:30:00 +0200
+tags: developers opensource spid sdk
+categories: news
+author: Francesco Pognante
+image: /assets/icons/logo-it.png
+locale: it
+
```
Comment (bfabio) [inline _posts/it/2021-07-30-community-call-14-luglio.md:11]:
```suggestion
---
```

File: _posts/it/2021-07-30-community-call-14-luglio.md [hunk 7]
```diff
@@ -0,0 +1,17 @@
...
+layout: post
+title: "“Risorse per l’integrazione del Sistema Pubblico di Identità Digitale"
```
Comment (bfabio) [inline _posts/it/2021-07-30-community-call-14-luglio.md:3]:
```suggestion
title: "Risorse per l’integrazione del Sistema Pubblico di Identità Digitale"
```

File: _posts/it/2021-07-30-community-call-14-luglio.md [hunk 8]
```diff
@@ -0,0 +1,17 @@
...
+A concludere l’incontro, una sessione di domande e risposte come sempre ricca di spunti, idee e riflessioni, grazie agli interventi dei partecipanti.
+Dopo la pausa estiva, la community si ritroverà il prossimo **22 settembre**. Per sapere il tema della prossima call **resta in contatto con la community su [Slack](https://developersitalia.slack.com/archives/C023MSRP03V)** e **segui [Developer Italia](https://twitter.com/developersITA) e [Designers Italia])https://twitter.com/DesignersITA) anche su Twitter**.
```
Comment (bfabio) [inline _posts/it/2021-07-30-community-call-14-luglio.md:16]:
```suggestion
Dopo la pausa estiva, la community si ritroverà il prossimo **22 settembre**. Per sapere il tema della prossima call **resta in contatto con la community su [Slack](https://developersitalia.slack.com/archives/C023MSRP03V)** e **segui [Developer Italia](https://twitter.com/developersITA) e [Designers Italia](https://twitter.com/DesignersITA) anche su Twitter**.
```

Comment (bfabio) [inline _posts/it/2021-07-30-community-call-14-luglio.md:16]:
```suggestion
Dopo la pausa estiva, la community si ritroverà il prossimo **22 settembre**. Per sapere il tema della prossima call **resta in contatto con la community su [Slack](https://developersitalia.slack.com/archives/C023MSRP03V){:target="_blank"}** e **segui [Developer Italia](https://twitter.com/developersITA){:target="_blank"} e [Designers Italia](https://twitter.com/DesignersITA){:target="_blank"} anche su Twitter**.
```

File: _posts/it/2021-07-30-community-call-14-luglio.md [hunk 9]
```diff
@@ -0,0 +1,21 @@
...
+
+A concludere l’incontro, una sessione di domande e risposte come sempre ricca di spunti, idee e riflessioni, grazie agli interventi dei partecipanti.
+
+Dopo la pausa estiva, la community si ritroverà il prossimo **22 settembre**. Per sapere il tema della prossima call **resta in contatto con la community su [Slack](https://developersitalia.slack.com/archives/C023MSRP03V){:target="_blank"}** e **segui [Developer Italia](https://twitter.com/developersITA){:target="_blank"} e [Designers Italia](https://twitter.com/DesignersITA){:target="_blank"} anche su Twitter**.
+
+**Se vuoi rivedere la community call del 14 luglio, [guarda la registrazione su YouTube])(https://www.youtube.com/watch?v=NoMUqVKK8Rc){:target="_blank"}.**
```
Comment (bfabio) [inline _posts/it/2021-07-30-community-call-14-luglio.md:21]:
```suggestion
**Se vuoi rivedere la community call del 14 luglio, [guarda la registrazione su YouTube](https://www.youtube.com/watch?v=NoMUqVKK8Rc){:target="_blank"}.** 
```

File: _posts/it/2021-07-30-community-call-14-luglio.md [hunk 10]
```diff
@@ -0,0 +1,21 @@
...
+
+Nel corso della call, **Davide Brunato** ci ha raccontato il suo lavoro su spid-django, una app che semplifica l’integrazione SPID nei progetti basati su Django Framework. **Michele d’Amico** ha poi descritto la sua esperienza con l’integrazione di SPID tramite SimpleSAMLphp, la sua partecipazione alla Hackathon Developers e la creazione di spid-php. A seguire, **Roberto Gallea** ci ha illustrato [spid-sp-shibboleth](https://github.com/italia/spid-sp-shibboleth){:target="_blank"} e [spid-idp-proxy-shibboleth](https://github.com/italia/spid-idp-proxy-shibboleth){:target="_blank"}, **Luca Leonardo Scorcia** ha descritto le peculiarità di  [spid-keycloak-provider](https://github.com/italia/spid-keycloak-provider){:target="_blank"} e, infine, **Daniele Giallonardo** ci ha spiegato quali sono le caratteristiche di  [spid-aspnetcore](https://github.com/italia/spid-aspnetcore){:target="_blank"}.
```
Comment (bfabio) [inline _posts/it/2021-07-30-community-call-14-luglio.md:15]:
```suggestion
Nel corso della call, **Davide Brunato** ci ha raccontato il suo lavoro su spid-django, una app che semplifica l’integrazione SPID nei progetti basati su Django Framework. **Michele d’Amico** ha poi descritto la sua esperienza con l’integrazione di SPID tramite SimpleSAMLphp, la sua partecipazione alla Hackathon Developers e la creazione di [spid-php](https://github.com/italia/spid-php){:target="_blank"}. A seguire, **Roberto Gallea** ci ha illustrato [spid-sp-shibboleth](https://github.com/italia/spid-sp-shibboleth){:target="_blank"} e [spid-idp-proxy-shibboleth](https://github.com/italia/spid-idp-proxy-shibboleth){:target="_blank"}, **Luca Leonardo Scorcia** ha descritto le peculiarità di  [spid-keycloak-provider](https://github.com/italia/spid-keycloak-provider){:target="_blank"} e, infine, **Daniele Giallonardo** ci ha spiegato quali sono le caratteristiche di  [spid-aspnetcore](https://github.com/italia/spid-aspnetcore){:target="_blank"}.
```

**Review by bfabio — APPROVED** [review]

**Pull request merged**